### DISCUSSION TITLE
Line tooltip

### DISCUSSION THREAD

**Discussion by sindhai-kathiravan** [themustafaomar/jsvectormap#72] [Q&A]
Hello Mustafa Omar,

Thank you for this jsvector library. I am verymuch impressed on this library. It is very helpful for my project. I am trying to add LineSelect event, removeLine and Line label and line tooltip. I have tried my level. But Unable to complete. can you please update the library with this feature. Thank will be helpful for all users. 

Thank you
Kathir

**Comment by themustafaomar** [comment]
Hi @sindhai-kathiravan 

Glad it helped!

I've just added a new method `removeLine` to remove lines programmatically and it'll be available on the next version.

As for line select, line label and line tooltip, actually I'm not planning to add them natively.

However, I'll try to provide the flexibility to add them out of the box with easy workarounds.

The reason why I don't want to support (for example: the line tooltip) is because the line is very thin, it's too hard to hover over it.
